doc/source/cluster/running-applications/job-submission/quickstart.rst

```diff
@@ -76,6 +76,12 @@ To tell the Ray Jobs CLI how to find your Ray Cluster, we will pass the Ray Dash
 
 Alternatively, you can also pass the ``--address=http://127.0.0.1:8265`` flag explicitly to each Ray Jobs CLI command, or prepend each command with ``RAY_ADDRESS=http://127.0.0.1:8265``.
 
+Additionally, if you wish to pass headers per HTTP request to the Cluster, then utilize the `RAY_JOB_HEADERS` environment variable.` Must be in JSON form.
+
+.. code-block:: bash
+
+    $ export RAY_JOB_HEADERS="{'Bearer': 'TOKEN'"}
+
```

Comment:
We'd like to keep the quickstart as minimal as possible, is there somewhere else in the docs it makes sense to include this snippet?

Reply:
I was doing some reading through the sections and noticed a file cli.rst. Would it make sense to include it here. I originally put it here because we do something very similar for RAY_ADDRESS in the quickstart file.

Reply:
I think that would make sense!